play/nix/shell.nix

```diff
@@ -14,14 +14,17 @@ stdenv.mkDerivation rec {
   buildInputs = with pkgs; [
       cmake
       rustup
+      openssl
+      pkg-config
       lld
       python38Packages.pip
     ];
 
   hardeningDisable = [ "fortify" ];
 
+  MZ_DEV = 1;
+  RUSTFLAGS = "-C link-arg=-fuse-ld=lld -C debuginfo=0";
```

Comment:
Hm, I'm pretty sure this will break for nix folks on darwin.

Reply:
I just moved this variable from below, it's not a change of this PR